src/domain/state.ts

```diff
@@ -652,6 +660,15 @@ function transitionToApproved(current: Application, updatePart: Partial<UpdateAp
   return current;
 }
 
+function transitionToClosed(current: Application) {
```

Comment:
is there a `closedBy` field that we could use to store who did the closure ?

Reply:
so while doing the reject state, there is no last updated by or closed, approved, rejected by
I ended up using the updates : [ ] field to store audit info like who did approve and reject.

Reply:
not currently but i can ask rosi and kim if they think this would be useful to track. certainly wouldn't hurt.

Reply:
ok. so, would that look like something like:
```{
details: 'closed by {userId}';
  type: 'app closed' ; // do we want to have specific types here?
  date: dateClosed;
}``` ?

Reply:
I updated the structure a bit https://github.com/icgc-argo/dac-api/pull/173/files

Reply:
oh i didn't realize there is an actual `closedBy` and `closedAtUtc` key in the application interface. i can use those instead of the updates list, if that sounds good

Reply:
ok cool yeah def use that it's sufficient